docs/sources/flow/concepts/config-language/_index.md

```diff
@@ -1,14 +1,14 @@
 ---
 aliases:
-- /docs/grafana-cloud/agent/flow/config-language/
-- /docs/grafana-cloud/monitor-infrastructure/agent/flow/config-language/
-- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/config-language/
-- /docs/grafana-cloud/send-data/agent/flow/config-language/
+- /docs/grafana-cloud/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/send-data/agent/flow/concepts/config-language/
 - configuration-language/
```

Comment:
All of the moved pages should have an alias to the old page so old links continue to work.

Reply:
Thanks for reminding me about this. Done now. 
I'm not sure what are the relative aliases for, like `- ../../configuration-language/expressions/function-calls/`. I've updated them to match their new nesting level and I've tested locally and I didn't find any broken links.

Reply:
Hm, Clayton is on vacation until the 8th so he wouldn't be able to help us right now. @jdbaldry do you happen to know?

Reply:
Note: these relative aliases were [added here](https://github.com/grafana/agent/pull/2868/files#diff-2567292308fa5c1221c47838c247dbbb7b8f39c1f2b0645cb9e4b324b54e8b6b) and appear to be a way to express an alias to the `latest` or the currently browsed version of the page. If I understand it right, just updating them to match their nesting depth should be the right thing to do here.